playground/ssr-resolve/pkg-exports/index.js

```diff
@@ -0,0 +1 @@
+module.exports = undefined
```

Comment:
Is this file required? It is not used currently.

Reply:
I'll leave it in so we have multiple entry points for this package.